client/src/components/SearchForm.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import PinIcon from "./PinIcon";
+import DotsIcon from "./DotsIcon";
+import Select from "./Search";
```

Comment:
What is "Select"? You use "Select" down in the rest of the code instead of Search. Rename Select to Search to avoid confusion.